### PR TITLE
Development

### PR DESCRIPTION
**Pull request by george-aidonidis**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - "6"
```
Comment (mariosant) [inline .travis.yml]: Why using version 6?

File: test/index.js [hunk 2]
```diff
@@ -11,10 +11,11 @@ db.addMiddleware(addTimestamps);
 describe('Middleware timestamps', () => {
 	let dummyCollection;
 	before(() => {
```
Comment (mariosant) [inline test/index.js]: This includes an async call `db.create` and it should wait for this to finish before proceeding.
Check out http://mochajs.org/#working-with-promises for handling it.

File: test/index.js [hunk 3]
```diff
@@ -11,10 +11,11 @@ db.addMiddleware(addTimestamps);
...
+		db.create('dummy');
 		dummyCollection = db.get('dummy');
 	});
 	after(() => {
```
Comment (mariosant) [inline test/index.js]: I am not sure if my comment about async handling applies here as well. Chances are it is.

**Review by mariosant — COMMENTED** [review]
A few comments about async handling.

File: .travis.yml (hunk 1 above)
Comment (george-aidonidis) [reply .travis.yml]: My bad. Let's use 7.10.1, sounds good?

**Comment by george-aidonidis** [comment]
@mariosant as you suggested, I think we should also return the promises inside the `it` functions, right?

**Review by mariosant — APPROVED** [review]
Superb work here, dude!

**Pull request merged**